packages/account/src/Components/api-token/__tests__/api-token-table-row-token-cell.spec.tsx

```diff
@@ -4,7 +4,7 @@ import userEvent from '@testing-library/user-event';
 import ApiTokenTableRowTokenCell from '../api-token-table-row-token-cell';
 
 describe('ApiTokenTableRowTokenCell', () => {
-    const mock_props = {
+    const mock_props: React.ComponentProps<typeof ApiTokenTableRowTokenCell> = {
```

Comment:
Same

Reply:
This is the usual convention that we use everywhere for types in test case files